elixir/apps/domain/lib/domain/resources/resource.ex

```diff
@@ -21,7 +21,9 @@ defmodule Domain.Resources.Resource do
 
     has_many :policies, Domain.Policies.Policy, where: [deleted_at: nil]
     has_many :actor_groups, through: [:policies, :actor_group]
-    field :authorized_by_policy, :map, virtual: true
+
+    # Warning: do not do Repo.
```

Comment:
Just for my own edification, what does this comment mean?

Reply:
Well, this means I was distracted while typing a comment, will push a fix :D